{"reason.description.step":"ステップで一時停止しました","reason.description.breakpoint":"ブレークポイントで一時停止しました","reason.description.exception":"例外で一時停止しました","reason.description.uncaughtException":"キャッチされない例外で一時停止しました","reason.description.caughtException":"キャッチした例外で一時停止しました","reason.description.user_request":"ユーザー要求で一時停止しました","reason.description.entry":"エントリで一時停止しました","reason.description.debugger_statement":"デバッガー ステートメントで一時停止しました","reason.description.restart":"フレーム エントリで一時停止しました","reason.description.promiseRejection":"Promise の拒否で一時停止しました"}